src/ServicePulse.Host/vue/src/composables/autoRefresh.ts

```diff
@@ -3,7 +3,7 @@ export default function useAutoRefresh(refreshAction: () => Promise<void>, timeo
 
   function stopTimer() {
     if (refreshInterval !== null) {
-      window.clearTimeout(refreshInterval);
+      clearTimeout(refreshInterval);
```

Comment:
this is an issue - setTimeout and clearTimeout are interpreted by typescript as the versions from Node rather than window

Reply:
What is the issue? Will that affect the bundle generated by the build? What am I missing?

Reply:
They have different signatures so TypeScript will complain about `setTimeout` but not `window.setTimeout`. The final bundle is unaffected. This is a develop/build-time issue.